src/elements/content-sidebar/activity-feed/activity-feed/ActivityFeed.scss

```diff
@@ -86,7 +86,7 @@
 
     .bcs-activity-feed-comment-input {
         background-color: $white;
-        flex-shrink: 0;
+        flex: 0 0 auto;
```

Comment:
Does the comment box still grow/shrink as expected given min/max height set below? It definitely looks like it's meant to change size.

Reply:
Yeah it does, the flex-grow: 0 specifies that it won't take up any of the remaining space in the flex container but will still reflect it's natural height